.github/workflows/auto-response.yml

```
@@ -1,29 +1,41 @@
 name: Issue/PR response
+
 permissions:
   issues: write
   pull-requests: write
+
 on:
   issues:
-    types:
-      - opened
+    types: [reopened, opened]
   pull_request_target:
-    types:
-      - opened
+    types: [reopened, opened]
```

...it wouldn't need to re-add to the project? That would surely be useless...?
```suggestion
    types: opened
```

---

As above.

---

See my comment above about changing the template in the org repo